react/components/SearchResult.js

```diff
@@ -8,7 +8,7 @@ import LoadingOverlay from './LoadingOverlay'
 import LayoutModeSwitcher from './LayoutModeSwitcher'
 import { searchResultPropTypes } from '../constants/propTypes'
 import OrderBy from './OrderBy'
-
+import { LAYOUT_MODE } from './LayoutModeSwitcher'
```

Comment:
`import { LAYOUT_MODE, LayoutModeSwitcher  } from './LayoutModeSwitcher'` and remove line 8